src/js/utils/computed-style.js

```diff
@@ -8,21 +8,19 @@ import window from 'global/window';
  * A safe getComputedStyle.
  *
  * This is needed because in Firefox, if the player is loaded in an iframe with
- * `display:none`, then `getComputedStyle` returns `null`, so, we do a null-check to
- * make sure  that the player doesn't break in these cases.
+ * `display:none`, then `getComputedStyle` returns `null`, so, we do a
+ * null-check to make sure that the player doesn't break in these cases.
  *
- * @param {Element} el
- *        The element you want the computed style of
+ * @function
+ * @param    {Element} el
+ *           The element you want the computed style of
  *
- * @param {string} prop
- *        The property name you want
+ * @param    {string} prop
+ *           The property name you want
  *
- * @see https://bugzilla.mozilla.org/show_bug.cgi?id=548397
- *
- * @static
- * @const
+ * @see      https://bugzilla.mozilla.org/show_bug.cgi?id=548397
  */
-export default function computedStyle(el, prop) {
+function computedStyle(el, prop) {
```

Comment:
This happens in a few places. Having `export default function name` seemed to confuse things when we got to the rendered docs. Moving the `export default` made it better.

Reply:
That sounds like a good idea, but I think it's better to have useful documentation.